docs/_drafts/scv2-hodometer-spartakus-container.mmd

```diff
@@ -0,0 +1,23 @@
+---
+title: Spartakus container architecture
+---
+
+flowchart TD
+  S["Volunteer\n\nMetrics publisher"]
+  C["Collector\n\nMetrics receiver"]
+  K["Kubernetes API"]
+
+  subgraph k8s-cluster[Kubernetes cluster]
+    S --> |"queries\n[HTTP]"| K
+  end
+  S ---> |"pushes metrics to\n[HTTP]"| C
+
+  classDef focusSystem      fill:#1168bd,stroke:#0b4884,color:#ffffff
+  classDef supportingSystem fill:#666,stroke:#0b4884,color:#ffffff
+  classDef consumingSystem  fill:#08427b,stroke:#052e56,color:#ffffff
+
+  class S focusSystem
+  class C consumingSystem
+  class K supportingSystem
+
+  style k8s-cluster fill:#eee,stroke:#ccc,stroke-width:2px,stroke-dasharray: 5 5
```

Comment:
:question: Can subgraph styles be turned into a `classDef`?  If so, that would slightly simplify the definitions by providing better grouping and supporting multiple subgraphs (if required in the future) without duplication

Reply:
The answer is a resounding (and fortunately unsurprising) yes.  Fixed in https://github.com/agrski/agrski.github.io/pull/40/commits/586cee78924a3b247ee4fe4734c9a81882f4849d